runtests.py

```diff
@@ -141,8 +141,8 @@ def import_refnanny():
     'tag:pstats':   'pstats',
     'tag:posix':    'posix',
     'tag:array':    'array',
-    'tag:coverage': 'Cython.Coverage',
-    'Coverage':     'Cython.Coverage',
+    'tag:coverage': ['Cython.Coverage', 'coverage.tracer'],  # coverage.tracer is the C tracer
+    'Coverage':     ['Cython.Coverage', 'coverage.tracer'],
```

Comment:
What do you think of letting the import of `Cython.Coverage` fail if we know that Cython coverage reporting isn't supported? If users request the plugin, but that can't provide the service, a failing import seems like an easy way to let users handle it on their side.

Reply:
Done.